src/AccessibilityInsights.Extensions.GitHub/IIssueFormatter.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public interface IIssueFormatter
```

Comment:
Please add comments here to explain what this interface is for and what the methods are for.

Reply:
The name should be descriptive enough but I'll add comments